flag-app/flag-app-project/src/CountriesContextProvider.tsx

```diff
@@ -4,12 +4,24 @@ import { useApiData } from "./hooks";
 
 const CountriesContextProvider: FC<PropsWithChildren> = ({ children }) => {
   const [searchInput, setSearchInput] = useState("");
-  const { data: countries, error, isLoading } = useApiData(searchInput);
+  const [filterValue, setFilterValue] = useState("Filter by Region");
```

Comment:
this default value "Filter by region" should not belong within a state containing user input. What you placed here is a text placeholder, so you should move it out and keep an **empty string** as the default value here.